### ISSUE TITLE
Angular Bootstrap typeahead repeats aria label upon selection

### ISSUE THREAD

**Issue by edcourtney74** [FreedomScientific/standards-support#769]
### Summary

When I tab into the Angular Bootstrap typeahead, the aria-label is announced as expected. But then when I make a selection, the aria-label is repeated again in its entirety (the label, instructions, etc.). 

### Expected result

After typeahead selection, expected JAWS to announce only the selection made. The aria label should not be repeated.

### Actual result

JAWS repeats the aria label, then the selection made and then the instructions.

### Example

https://ng-bootstrap.github.io/#/components/typeahead/examples

On the Simple Typeahead example, after a selection is made, "Search for a state" is repeated.

#### JAWS version and build number
JAWS 2023.2212.23

#### Operating System and version
Windows 10

#### Browser and version:
Google Chrome Version 117.0.5938.89

**Comment by JAWS-test** [comment]
The problem occurs with JAWS and NVDA.

It occurs with comboboxes that are not implemented with aria-activedescendant, i.e. whenever the focus actually goes into the list and does not remain in the input field. The cause is that the combobox is left and then receives focus again - the screen readers cannot recognise that the focused input field was the triggering element and does not have to be output again. 

* combobox without aria-activedescendant, where the problem also occurs: https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-datepicker/
* combobox with aria-activedescendant, where the problem does not occur: https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-autocomplete-both/

**Comment by edcourtney74** [comment]
Thanks for the help
